.gitignore

```diff
@@ -5,6 +5,8 @@
 .idea/vcs.xml
 .idea/jsLibraryMappings.xml
 *.iml
+
```

Comment:
chromedriver.exe still in repo

Reply:
![image](https://user-images.githubusercontent.com/32294404/141378476-d9dc5d88-9e5f-42a0-a5e6-c589363744ca.png)
It was always declared in gitignore